apps/nextjs-website/src/app/[[...slug]]/layout.tsx

```diff
@@ -139,7 +139,11 @@ export default async function Layout({
               defaultLocale={defaultLocale}
             />
           )}
-          <Header {...headerProps} />
+          <Header
+            {...headerProps}
+            locales={localesArray as Array<Locale>}
```

Comment:
```suggestion
            locale={locale}
```
This is literally never used, just pass the current locale.